Deep Investigations/Cloud solutions/Details.md

```diff
@@ -0,0 +1,180 @@
+# Nextcloud
+
+## Table of contents
+
+- [Introduction](#introduction)
+    - [Why we choose](#why-we-choose)
+    - [Learing Outcomes](#learning-outcomes)
+- [History](#history)
+- [Installation](#installation)
+    - [System requirements](#system-requirements)
+        - [Server](#server)
+        - [Desktop Client](#desktop-client)
+        - [Web Application](#web-application)
+    - [Prerequisites for manual installation](#prerequisites-for-manual-installation)
+    - [Apache Web server configuration](#apache-web-server-configuration)
+    - [Installation on Ubuntu 20.04 LTS](#installation-on-ubuntu-20.04-lts)
+- [FAQ](#faq)
+- [References](#references)
+
+##  Introduction
+---
+
+Nextcloud is a fully featured on-premises solution to share and collaborate documents, send and receive emails, manage all audio/video conferencing, manage contacts, manage schedules, etc. Nextcloud is scalable from home office solutions to full sized data center solutions that support millions of users.
+
+Nextcloud permits user and group administration via [LDAP](https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol) or [OpenID](https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol). Content can be shared by defining granular read/write permissions between users and groups. Alternatively, Nextcloud users can create public URLs when sharing files. Logging of file-related actions and disallowing access based on file access rules are also available.
+
+Nextcloud can be extended with plugins to add extra functionalities to the system. The **Nextcloud App store** can be found [here](https://apps.nextcloud.com/), and it contains **more than 200 plugins**. Some of the functionalities that you can add to Nextcloud are as follows.
```

Comment:
Add plugin installation to demo

Reply:
Sure ayye, I'll add this to the Course content demo videos list.